src/modules/Dashboard/features/RespondentData/RespondentData.styles.ts

```diff
@@ -23,3 +23,13 @@ export const StyledTextBtn = styled(Button)`
     fill: ${variables.palette.primary};
   }
 `;
+
+export const StyledButton = styled(Button)`
+  min-width: 10rem;
+  padding: ${theme.spacing(0)};
+  color: ${variables.palette.on_secondary_container};
+`;
+
+export const ActionButton = styled(StyledButton)`
+  padding: ${theme.spacing(1.4, 2.4)};
+`;
```

Comment:
I don't think we need any custom button styled components here after your latest changes. See next suggestion 👇🏻